src/hooks/index.ts

```diff
@@ -13,8 +9,14 @@ sbvrUtils.addPureHook('all', 'all', 'all', {
 	},
 });
 
-readdirAsync(__dirname + '/resources').each(initScript => {
-	if (/\.ts$/.test(initScript)) {
-		return require(__dirname + '/resources/' + initScript);
-	}
-});
+import('./resources/api_key');
```

Comment:
Any reason for these being async imports?  It looks like they could just be imported directly since we're at the top level